lib/ssz_ex.ex

```diff
@@ -8,23 +8,29 @@ defmodule LambdaEthereumConsensus.SszEx do
   #################
   ### Public API
   #################
+  import Bitwise
 
   @bits_per_chunk 256
+  @bytes_per_chunk 32
+  @bits_per_byte 8
```

Comment:
We should compute `@bits_per_chunk` by multiplying `@bits_per_byte` and `@bytes_per_chunk`